go/vt/sqlparser/sql.y

```diff
@@ -9150,6 +9166,12 @@ column_name_safe_keyword:
 | MIN
 | SUM
 
+// Keywords that aren't reserved or non-reserved in mysql, but are labeled as keywords in our grammar because of their special syntax.
```

Comment:
```suggestion
// Names of functions that require special grammar support. These aren't reserved or non-reserved in MySQL's docs, but are labeled as keywords in our grammar because of their custom syntax.
```